tools/depends/target/Makefile

```diff
@@ -57,7 +57,8 @@ DEPENDS = \
   taglib \
   tinyxml \
   udfread \
-  xz
+  xz \
+  date
```

Comment:
These needs to be changed to alphabetical sorting

Reply:
Done